packages/eslint-plugin/src/rules/no-duplicate-type-constituents.ts

```diff
@@ -5,13 +5,10 @@ import type { Type } from 'typescript';
 import { createRule, getParserServices } from '../util';
 
 export type Options = [
-  {
-    ignoreIntersections?: boolean;
-    ignoreUnions?: boolean;
-  },
+  { ignoreIntersections?: boolean; ignoreUnions?: boolean },
```

Comment:
These changes don't seem to be within the scope of this PR. Could you please revert them? Unnecessary changes make the reviewer's job harder :(

Reply:
yep! Sorry about that. Wish Prettier was more strict haha